demo/pdfpc-demo.tex

```diff
@@ -28,6 +28,7 @@
 \mode<presentation>
 
 \title{\pdfpc demo}
+\subtitle{(for v4.5.0)}
```

Comment:
I think this is not needed since the demo slides are always generated for the "right" version. In this case the changes live in master which will be the next v4.5

Reply:
> I think this is not needed since the demo slides are always generated for the "right" version. In this case the changes live in master which will be the next v4.5

That's right, but then the document gets published on our website, people google for it and become confused, as in #526...